.virtualenv.requirements.txt

```diff
@@ -18,4 +18,4 @@ cerberus
 kiwi
 
 # download with progress info
-wget
+progressbar
```

Comment:
`progressbar2` is the maintained, API-compatible fork of `progressbar`. Use that instead.

Reply:
good point, tested and done